examples/runcards/rqs.yml

```diff
@@ -0,0 +1,33 @@
+# Lattice
+lattice_length: 6
+lattice_dimension: 2
+
+# Target
+target: phi_four
+m_sq: -4
+lam: 1.0
+use_arxiv_version: false
```

Comment:
this example is not up to date - do we want to keep it?

Reply:
no